icecandidate.go

```diff
@@ -8,6 +8,7 @@ import (
 
 // ICECandidate represents a ice candidate
 type ICECandidate struct {
+	ObjectID       string           `json:"objectID"`
```

Comment:
Is it possible to not export `ObjectID` (looking at the RFC I don't see this key mentioned)